### ISSUE TITLE
License & Acknowledgements Screen

### ISSUE THREAD

**Issue by felixrieseberg** [TryGhost/Ghost-Desktop#179]
Would be nice to have a small screen that displays a) the packages we used and b) how the app is licensed.

This could be part of either the "About Ghost" screen - or somewhere under help.

**Comment by bnookala** [comment]
Do we want something similar to the chrome credits screen? ie chrome://credits/

**Comment by felixrieseberg** [comment]
Yes, but less ugly (and not in the main view, but in a separate BrowserWindow). GitHub's app is pretty basic in this, but they have something that looks like this:

![screen shot 2016-08-01 at 1 45 43 pm](https://cloud.githubusercontent.com/assets/1426799/17308345/4cf5f332-57ee-11e6-9eac-e2199b8edcd2.png)

I'm thinking that we generate something at build time (license plus used packages) and display it in a decent way ;-)

**Comment by bnookala** [comment]
@felixrieseberg I feel like I can pull something like this together :)
